BUILD

```diff
@@ -0,0 +1,13 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_prefix")
```

Comment:
Is the hack script in core decent?  Having the bazel build files updated automatically would be nice.